docs/releases/3.2.txt

```diff
@@ -95,6 +95,13 @@ or on a per-model basis::
 In anticipation of the changing default, a system check will provide a warning
 if you do not have an explicit setting for :setting:`DEFAULT_AUTO_FIELD`.
 
+.. _new_functional_indexes:
+
+Functional indexes
+------------------
+
+... TODO
```

Comment:
TODO

Reply:
I left this TODO. I'm going to continue working on this patch in the next week.